README.md

```diff
@@ -7,16 +7,18 @@
 ```python
 print('what to print')
 ```
+- output :<br> 
+what to print
 
 ### How to write comments
 
-#### Single Line
+### Single Line
 
 ```python
 # this is a single line comment.
 ```
 
-#### Multi Line
+### Multi Line
```

Comment:
Secara hierarki harusny header "Single Line" dan "Multi Line" itu bagian dari "How to write comments", jadi ukuranny harusny lebih kecil. Dk perlu di ubah lg.

kalo kw nulis gini di markdown:
```
## header
#### header juga
###### header lagi
```
Hasilny bakal gini:

## header
#### header juga
###### header lagi